src/DataGrid.tsx

```diff
@@ -690,6 +690,7 @@ function DataGrid<R, SR, K extends Key>(
   function isCellEditable(position: Position): boolean {
     return (
       isCellWithinViewportBounds(position) &&
+      position.idx !== -1 &&
```

Comment:
`isCellWithinViewportBounds` doesn't catch this? 🤔 
Should we add a test?

Reply:
It uses https://github.com/adazzle/react-data-grid/blob/main/src/DataGrid.tsx#L674 and `minColIdx === -1`

Reply:
May be we need to add a new function `withinEditBounds`

Reply:
Not sure how to test it. I will check